prerendering-same-site.md

```diff
@@ -90,7 +92,7 @@ The [speculation rules API](./triggers.md) can be used to trigger a prerender to
 <a href="https://a.test/foo">Click me!</a>
 ```
 
-The `<script type="speculationrules">` block here hints to prerender `https://a.test/foo`. As in the previous section, such prerendering includes sending the `Sec-Purpose` header.
+The `<script type="speculationrules">` block here hints to prerender `https://a.test/foo`. As in the previous section, such prerendering includes sending the `Sec-Purpose` header. If the prerendered URL (or ultimate response URL, after redirects) is cross-site, then prerendering will fail. If it is same-site but cross-origin, then the destination needs the opt-in [`Supports-Loading-Mode: credentialed-prerender` header](./opt-in.md#cross-origin-same-site-prerendering).
```

Comment:
For the first sentence, I would expect the cross-site URL restriction to apply to not just the first and last URLs in the redirect chain, but all of them. (Of course, it's probably uncommon to redirect cross-site and back again, but I live for edge cases.)

Reply:
Yeah I can see how this is confusing. Let me delete the parenthetical, so we just talk about "prerendered URL" and ignore redirects in this section. Discussion of them can then be done at leisure in the dedicated cross-origin same-site section below.